components/user_settings/advanced/user_settings_advanced.tsx

```diff
@@ -448,7 +476,7 @@ export default class AdvancedSettingsDisplay extends React.PureComponent {
         );
     }
 
-    renderFeatureLabel(feature) {
+    renderFeatureLabel(feature: string): JSX.Element | null {
```

Comment:
Similarly, most of these can be `React.ReactNode` as well

Reply:
Yep, you are right. Updated